packages/quantic/cypress/integration/facets/facet-common-selectors.ts

```diff
@@ -0,0 +1,31 @@
+import {ComponentSelector} from '../common-selectors';
+
+export interface BaseFacetSelector extends ComponentSelector {
+  label: () => Cypress.Chainable<JQuery<HTMLElement>>;
```

Comment:
would it be worthwhile creating a shorthand type for `Cypress.Chainable<JQuery<HTMLElement>>`? It would only serve to clean up the visual aspect of a few files but it's repeated a lot so maybe it's worth considering.

Reply:
Good point indeed!